ui/hooks/useGasFeeInputs.js

```diff
@@ -200,8 +212,8 @@ export function useGasFeeInputs(defaultEstimateToUse = 'medium', transaction) {
       : null,
   );
   const [gasLimit, setGasLimit] = useState(
-    transaction?.txParams?.gas
-      ? Number(hexToDecimal(transaction.txParams.gas))
```

Comment:
Hmm... this won't work for standard transactions in metamask. `gasLimit` is normally called `gas`

Instead could you call the param passed in from "View-quote" `gas`? (Then I think this change won't be necessary

Reply:
Also, we need to keep that `Number` conversion present to handle transactions in the standard metamask send flow

Could you convert the value to hex in `View-quote`? You can use a function called `hexToDecimal` for that

Reply:
I realize I might have been the original author of these changes :sweat_smile:

Reply:
Will do the suggested changes.

Reply:
Done. Btw, on the View Quote page `maxGasLimit` is already in hex, so I can just do this:
![image](https://user-images.githubusercontent.com/80175477/127546113-67f0cc1d-77b8-443c-a9b8-bd7cf1c9a253.png)

Edit Gas modal after these changes:
![image](https://user-images.githubusercontent.com/80175477/127545980-436d7a73-877f-456d-9fde-00e1c59bc137.png)